### ISSUE TITLE
CREATE EXTENSION propagation does not escape extension name

### ISSUE THREAD

**Issue by marcocitus** [citusdata/citus#3216]
Saw the following error:
```sql
2019-11-22 14:58:07.288 UTC [97] ERROR:  syntax error at or near "-"
2019-11-22 14:58:07.288 UTC [97] CONTEXT:  while executing command on 10.0.0.6:5432
2019-11-22 14:58:07.288 UTC [97] STATEMENT:  SELECT * from master_add_node($1, $2)
```

Caused by not escaping the extension name in the new extension propagation feature:
```sql
2019-11-22 15:05:35.963 UTC [153] ERROR:  syntax error at or near "-" at character 36
2019-11-22 15:05:35.963 UTC [153] STATEMENT:  CREATE EXTENSION IF NOT EXISTS uuid-ossp WITH SCHEMA public ;
```

**Comment by serprex** [comment]
There's more escaping issues, such as use of `regprocedureout` in `GetFunctionAlterOwnerCommand`. Will look into

**Comment by metdos** [comment]
@onurctirtir did we fix this with #3218? If yes, could you close the issue?

**Comment by onurctirtir** [comment]
Fixed in #3218 hence closing
